.github/workflows/reviewdog.yml

```
@@ -22,7 +22,25 @@ jobs:
           reporter: github-pr-review
           level: error
           fail_on_error: true
-          pattern: TODO_IN_THIS_
+          pattern: TODO_IN_THIS_|TODO_THIS_
+
+  # Makes sure that comments like TODO_UPNEXT prevent merging if there is
+  # no associated username right after.
+  #   Example failure: TODO_UPNEXT: <do something>
+  #   Example non-failure: TODO_UPNEXT(@olshansk): <do something>
+  # See https://github.com/pokt-network/action-fail-on-found
+  check_todo_upnext:
+    name: Check TODO_UPNEXT
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pokt-network/action-fail-on-found@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          level: error
+          fail_on_error: true
+          pattern: TODO_UPNEXT:
```

```suggestion
          pattern: "TODO_UPNEXT:"
```
This line breaks the config: https://github.com/pokt-network/poktroll/actions/runs/11561203740/workflow

With this suggestion committed, I think this should work.